addons/ekco/template/base/install.sh

```diff
@@ -336,7 +337,7 @@ function ekco_create_deployment() {
     cp "$src/rotate-certs-rbac.yaml" "$dst/rotate-certs-rbac.yaml"
 
     # is rook enabled
-    if kubectl get ns | grep -q rook-ceph; then
+    if [ -n "$ROOK_VERSION" ]; then
```

Comment:
is there any drawback to changing this?

Reply:
if someone used to have rook in the spec and so it's installed, but it's not in the most recent spec?